src/api/routes/v1/validator.ts

```diff
@@ -96,8 +96,7 @@ async function getValidators(): Promise<ValidatorResponse[]> {
       'master_key',
       'revoked',
     ])
-    .whereNotNull('signing_key')
-    .orWhereNotNull('master_key')
+    .where('revoked','=','false')
```

Comment:
@natenichols, see here re: checking for revoked validators. I believe we initially decided to display all validators even if they are revoked. But, since we only delete validators from the database after 7 days of inactivity, I thought it is best if we just don't display the revoked ones. On the livenet explorer, you can see that Bithomp is listed twice because the entry for the old (revoked) signing key is still in the validators table and won't get removed for 7 days. Alternatively, we could just modify the explorer to not display the revoked validators.

Reply:
Yup, that sounds good to me!